theme/default/js/scripts.js

```diff
@@ -0,0 +1,15 @@
+$(document).ready(function() {
```

Comment:
What does this do?

Should this be in English and served through Xibo's translations?